packages/alpha/src/__tests__/api/simulatorRoutinesApi.unit.spec.ts

```diff
@@ -46,6 +47,12 @@ describe('Simulator Routines API unit tests', () => {
     });
     expect(result.items).toEqual(response.items);
     expect(result.items.length).toBe(2);
+    expect(result.items[0].simulatorIntegrationExternalId).toBe(
+      'integration-1'
+    );
+    expect(result.items[1].simulatorIntegrationExternalId).toBe(
+      'integration-2'
+    );
```

Comment:
```suggestion
expect(result.items).toEqual(response.items);
expect(result.items.length).toBe(2);
```

there is already `expect(result.items).toEqual(response.items);` that covers it

Reply:
https://github.com/cognitedata/cognite-sdk-js/pull/1346/commits/4bf2b3554492650d6de582130af8e48256664306